src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -60,6 +32,34 @@ const AccordionPanel = ({
     theme,
   );
 
+  const onHandleMouseOver = () => (...args) => {
```

Comment:
may i suggest to remove the () and in the caller to just add `onHandleMouseOver` seems a tiny bit more readable. same case with the rest of them. thanks for taking care of the changes.

Reply:
Fixed